src/main/java/com/GotYourBac/gotYourBac/controllers/HomeController.java

```diff
@@ -0,0 +1,24 @@
+package com.GotYourBac.gotYourBac.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class HomeController {
+
+    //Apparently GetMapping without the route will still give us what we want.
+    @GetMapping("/")
+    public String gethome(){
+        return "home";
+    }
+
+    @GetMapping("/balmerpeekinfo")
```

Comment:
A small nitpick: your routes should be camelcase or dashed, this is something users will see. Also this is the balmer peak, not peek

Reply:
Fixed camelCase